<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConverterWgt</class>
 <widget class="QWidget" name="ConverterWgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>121</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="2">
      <widget class="QComboBox" name="cmbCurrencyOut"/>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="ledValueIn"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="ledValueOut"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblOut">
       <property name="text">
        <string>Выходная сумма:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lblIn">
       <property name="text">
        <string>Входная сумма:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QComboBox" name="cmbCurrencyIn"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnSet">
       <property name="text">
        <string>Настройки</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>Закрыть</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
